lib/fluent/plugin/in_http.rb

```diff
@@ -533,7 +543,14 @@ def on_message_complete
         end
         path_info = uri.path
 
+        query_params = WEBrick::HTTPUtils.parse_query(uri.query)
+
+        query_params.each_pair {|k,v|
+          params["QUERY_#{k.gsub('-','_').upcase}"] = v
+        }
```

Comment:
Adding `if @add_query_params` is better.
Pass `@add_query_params` to `Handler`.

Reply:
Sure, added!